.gitignore

```diff
@@ -3,5 +3,6 @@
 # All directories (sub-repos)
 /*/
 
-# Explicitly keep lib directory
+# Explicitly keep lib and .github directory
 !/lib
+!/.github
```

Comment:
Please make sure to always end files with an EOL. This is easiest with an `.editorconfig` at the top level directory of all projects we work on together, in combination with the VS Code add-in.